aten/src/ATen/native/VariableMethodStubs.cpp

```diff
@@ -8,7 +8,7 @@
 namespace at {
 namespace native {
 
-void backward(const Tensor& self, const Tensor& gradient, c10::optional<bool> keep_graph, bool create_graph) {
+void backward(const Tensor& self, const Tensor& gradient, c10::optional<bool> keep_graph, bool create_graph, const Tensor& inputs) {
```

Comment:
As mentioned in the other comment I do think that you want a collection of Tensors here, not just a single one.

Reply:
Pretty sure this change depends on our declaration native_functions entry - so leaving this here for now until we get that figured out